frontend/src/static/js/main.js

```diff
@@ -164,6 +203,26 @@ window.app = new window.Vue({
     generateKey(dataObj) {
       return JSON.stringify(dataObj);
     },
+
+    hasCommits(info) {
+      return window.REPOS[info.repo].commits.authorFinalContributionMap[info.author] > 0;
+    },
+
+    removeInfoHash(hashObj) {
+      const tabKeys = ['tabAuthor', 'tabLocation'];
+      return hashObj.filter((value) => {
+        const key = value.split('=')[0];
+        return !tabKeys.includes(key);
+      });
+    },
+
+    addKeyToHash(params, key, val) {
+      const param = `${key}=${val}`;
+      if (!params.includes(param)) {
+        params.push(param);
+      }
+      return params;
+    },
```

Comment:
use `window.addHash` instead, since its already there, for whatever use case you need it for

Reply:
Similarly, mentioned below.